contributors/devel/sig-scheduling/scheduling_code_hierarchy_overview.md

```diff
@@ -66,6 +66,22 @@ Consists of the following four steps ran in the same order:
 In case of processing of any of the extension points fails, `Unreserve` operation
 of all `Reserve` plugins is invoked (e.g. free resources allocated for a gang of pods).
 
+## Scheduler extensibility
+
+The scheduler is extensible: the cluster administrator can choose which of the pre-defined scheduling policies to apply, and can add new ones.
+
+### Modifying policies
```

Comment:
This is actually gone now. I would say just remove it for now. The new API is mostly documented in the website:

https://kubernetes.io/docs/reference/scheduling/config/
https://kubernetes.io/docs/reference/config-api/kube-scheduler-config.v1beta2/

And the implementation details of these are already briefly mentioned in `Initial startup configuration` below